### PR TITLE
autoresize on switchTo

### PR DESCRIPTION
**Pull request by sandlerr**
/cc @zendesk/vegemite

The arbitrary max-height is just the height that works best on my laptop screen so that a single app almost fills the sidebar.

### CODE REVIEW

File: src/javascripts/base_app.js [hunk 1]
```diff
@@ -95,6 +97,8 @@ BaseApp.prototype = {
 
   switchTo: function(name, data) {
     this.$('[data-main]').html(this.renderTemplate(name, data));
+    var newHeight = Math.min($('html').height(), maxHeight);
+    this.zafClient.invoke('resize', { height: newHeight, width: '100%' });
```
Comment (danielbreves) [inline src/javascripts/base_app.js]: Shouldn't this be optional? What if they want the app to be smaller than the content?

Comment (sandlerr) [inline src/javascripts/base_app.js]: That's an unusual case and does not mirror the behaviour of legacy apps where the container div will always expand.
If that behaviour is needed developers should create a container element in their template and give it an appropriate `overflow` style using css.

Comment (sandlerr) [inline src/javascripts/base_app.js]: we might also want to patch this behaviour into jquery functions that manipulate the dom...

Comment (svizzari) [inline src/javascripts/base_app.js]: Proxy jQuery's `$(window).resize(() => {})` on to `ZAFClient.invoke('resize', ...)`?

**Pull request merged**